arenadata/Dockerfile

```diff
@@ -5,7 +5,11 @@ ARG sigar_headers=http://downloads.adsw.io/ADB/6.22.0_arenadata38/centos/7/commu
 
 # Reinstall glibc-common. This is necessary to get langpacks in docker
 # because docker images don't contain them.
-RUN sed -i 's/\(override_install_langs*\)/# \1/' /etc/yum.conf && \
+# Change URL for repos, because Centos 7 EOL 30.06.2024
+RUN sed -i 's|mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra|baseurl=http://mirror.adsw.io/centos/$releasever/os/$basearch|g' /etc/yum.repos.d/CentOS-Base.repo && \
```

Comment:
File arenadata/Dockerfile is public available while resource mirror.adsw.io is not!